spec/v1/envelope.md

```diff
@@ -7,7 +7,9 @@ authentication and serialization.
 
 ## Schema
 
-The format and protocol are defined per [DSSE v1.0].
+The RECOMMENDED format and protocol for Envelopes are defined per [DSSE v1.0].
+Producers MAY use other signature methods and formats so long as they meet
+the [Bundle] data structure requirements.
 
 ## Fields
```

Comment:
Envelopes can now be ~anything, yet this section is just talking about DSSE.

Do we expect all envelopes, DSSE or not, to have these fields?

Reply:
Yeah that makes sense to me.  Can definitely keep the +dsse advice though!

Reply:
Agreed, the mediaType advice should stay. We can track the discussion on multi-envelope types in #21.